dokka-runners/dokkatoo/buildSrc/build.gradle.kts

```diff
@@ -14,6 +14,6 @@ dependencies {
 
 java {
   toolchain {
-    languageVersion.set(JavaLanguageVersion.of(11))
+    languageVersion = JavaLanguageVersion.of(11)
```

Comment:
This version of Dokkatoo isn't connected to anything at the moment, and probably won't be worked on for a while. While editing it won't hurt, it also isn't necessary, so can you remove all of the changes in the `dokka-runners/dokkatoo` dir please?